doc/manual/local.mk

```diff
@@ -20,7 +20,7 @@ dummy-env = env -i \
 	NIX_SSL_CERT_FILE=/dummy/no-ca-bundle.crt \
 	NIX_STATE_DIR=/dummy
 
-nix-eval = $(dummy-env) $(bindir)/nix eval --experimental-features nix-command -I nix/corepkgs=corepkgs --store dummy:// --impure --raw
+nix-eval = $(dummy-env) $(bindir)/nix --experimental-features nix-command --store dummy:// eval -I nix/corepkgs=corepkgs --impure --raw
```

Comment:
```suggestion
nix-eval = $(dummy-env) $(bindir)/nix eval --experimental-features nix-command -I nix/corepkgs=corepkgs --store dummy:// --impure --raw
```

Reply:
This change was removed when removing the deprecation.